docs/docs/concepts/run-groups.md

```diff
@@ -0,0 +1,158 @@
+
+:::note 
+You can asynchronously run multiple tests from a script in one run group and wait for all of them to finish before deciding whether to deploy a new release.
+:::
+
+Run Groups track the status of all test suites and test runs to yield a single result.
+
+Users are allowed to attach more runs to a run group by adding them in the run information that’s available in the CLI.
+
+[Integrations](https://docs.tracetest.io/tools-and-integrations/overview) support this feature by creating a single run group for every execution.
+- [k6](https://docs.tracetest.io/tools-and-integrations/k6)
+- [Playwright](https://docs.tracetest.io/tools-and-integrations/playwright)
+- [Cypress](https://docs.tracetest.io/tools-and-integrations/cypress)
+- [Artillery](https://docs.tracetest.io/tools-and-integrations/artillery-plugin)
+
+Run groups are essential for integrations! Without it, a k6 test could potentially generate thousands of test runs that would pollute the environment and make it almost impossible to search for test runs not associated with that k6 test run.
+
+Another good use case for run groups is for CI runs: usually we don’t run a single test or a single test suite to test our systems. You can expect some users to have dozens of suites being run during a CI pipeline. With run groups, all the suite runs from the same pipeline will be shown on the same screen, making it easier for users to understand what failed. 
+
+Last, but not least important, if multiple tests are run inside a run group, they will run in parallel and finish faster than if you use a test suite to organize your tests. (Test suites run sequentially, so for the next test to be run, the current test must finish executing, which can be pretty time consuming.)
+
+## Using Run Groups from the CLI
+
+The way to use run groups from the CLI is by running multiple folders at once:
+
+![Run Groups CLI](../img/run-groups-cli.png)
+
+![Run Groups CLI 2](../img/run-groups-cli-2.png)
+
+Users can also manually wait for a run group to be done from the CLI.
+
+The above screenshots show that now you can run multiple tests at once by providing multiple `-f <file.yaml` parameters to the `tracetest run test` command. All those tests will be placed under the same run group.
+
+Normally, if you run two tests separately, both test runs would be part of different run groups:
+
+```
+tracetest run test -f test1.yaml
+```
+
+```
+tracetest run test -f test2.yaml
+```
+
+Now you also can specify the run group id when running a test. This would make the test part of the run group.
+
+```
+tracetest run test -f test1.yaml --group my-pipeline
+tracetest run test -f test2.yaml --group my-pipeline
+```
+
+With the `--group` argument, both tests would be run in parallel but would be seen inside the same run group.
```

Comment:
I asked the team about this in slack. I would expect you still need the --skip-result-wait flag in order for the second test to process before tthe first test is totally finished, but perhaps setting the -group flag implicitly means 'and dont wait' (which might make sense).